app/views/content/life-as-a-teacher/teaching-as-a-career.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Teaching as a career"
+heading: "Teaching as a career"
+category: Life as a teacher
```

Comment:
The `Pay and benefits` page and the `Explore subjects` page both have an image in the header (as a result of this PR) and then don't have images on the cards

The new `Teaching as a career` page doesn't have an image in the header but does have images on the top card

Is this deliberate? Should they be consistent? @jenhadfield-dfe

Reply:
I deliberately used the image in the header for Pay and Benefits as when you only have 2 cards, it looks strange with one with an image and one without. In this PR we're moving the career ladder page from P+B so it drops it from 3 nav cards to 2. 

On the subjects page, we decided when doing the PR that it looked strange to have one subject in focus with an image but not the others - we felt they should all be equally weighted. 

I wonder if it now looks strange to not be consistent across all subcategory pages? @jenhadfield-dfe what do you think?

Reply:
ah that makes sense - but I think we should also consider that more users will be on mobile, so we shouldn't necessarily make decisions just looking at desktop?